features/currencies/currency_management.feature

```diff
@@ -71,3 +71,15 @@ Feature: Currency management
         When I click "Disable" near "VEF"
         Then I should see disabled currency with code "VEF" in the list
         And I should see "Currency has been successfully disabled"
+
+    Scenario: Cannot update currency code
+        When I am editing currency with code "USD"
+        Then The code field should be disabled
```

Comment:
If yes, we should not :) We start with capital "I" and it is correct, due to english grammar rules. However, other words should be wrote down with lowercase (as i.e. ``Given store has default configuration``).
Moreover, in more abstract way of thinking, we always **starts** sentence with capital letter, because keyword is part of the sentence! :smile: Making behat scenarios more natural means also thinking about them as natural language, I suppose :)